index.js

```diff
@@ -4702,9 +4703,38 @@ export async function createCsharpBom(path, options) {
         pkgData = pkgData.slice(1);
       }
       const dlist = parseCsPkgData(pkgData);
+      const deps = dlist;
       if (dlist?.length) {
         pkgList = pkgList.concat(dlist);
       }
+
+      if (dlist?.length) {
+        for (const p of dlist) {
+          parentDependsOn.add(p);
+        }
+      }
+    }
+    if (parentDependsOn.size) {
+      const arrrr =  new Set();
+      const prefix = parentComponent["bom-ref"].split("/")[0];
+      parentDependsOn.forEach((dependsOn) => {
+        if(dependsOn.name && dependsOn.version){ 
+          //console.log("prefix: ",prefix);
+          let dependcy = `${prefix}/`;
```

Comment:
All the below lines can be combined to using template strings. This way we don't have to append strings together.